Tests/SwiftFormatPrettyPrintTests/IfStmtTests.swift

```diff
@@ -515,4 +515,49 @@ final class IfStmtTests: PrettyPrintTestCase {
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 50)
   }
+
+  func testMultipleIfStmts() {
+    let input =
+      """
+      if foo && bar { baz() } else if bar { baz() } else if foo { baz() } else { blargh() }
+      if foo && bar && quxxe { baz() } else if bar { baz() } else if foo { baz() } else if quxxe { baz() } else { blargh() }
+      if let foo = getmyfoo(), let bar = getmybar(), foo.baz && bar.baz { foo() } else { bar() }
+      if let foo = getmyfoo(), let bar = getmybar(), foo.baz && bar.baz && someOtherCondition { foo() } else { bar() }
+      if let foo = getmyfoo(), let bar = getmybar(), foo.baz && bar.baz && someOtherCondition { foo() }
+      """
+
+    let expected =
+      """
+      if foo && bar { baz() } else if bar { baz() } else if foo { baz() } else { blargh() }
+      if foo && bar && quxxe {
+        baz()
+      } else if bar {
+        baz()
+      } else if foo {
+        baz()
+      } else if quxxe {
+        baz()
+      } else {
+        blargh()
+      }
+      if let foo = getmyfoo(), let bar = getmybar(), foo.baz && bar.baz {
+        foo()
+      } else {
+        bar()
+      }
+      if let foo = getmyfoo(), let bar = getmybar(),
+        foo.baz && bar.baz && someOtherCondition
+      {
+        foo()
+      } else {
+        bar()
+      }
+      if let foo = getmyfoo(), let bar = getmybar(),
+        foo.baz && bar.baz && someOtherCondition
+      { foo() }
```

Comment:
I think it would be nice to have these breaks fire too when the condition wraps, if possible—that way, the body is only condensed to one line if the *whole statement* fits on one line. I'm surprised that doesn't happen already, since I'd expect the reset break that did fire to do so consistently with the breaks inside the braces. What's going on there, and would it be a lot harder to make it do that?

Reply:
I intentionally only added the consistent group when there is an else clause, since the problem of inconsistent breaks around the clauses don't exist when there's only 1 clause. It's a simple change to unconditionally add the consistent group though.